src/python/visclaw/data.py

```diff
@@ -621,12 +622,17 @@ def __init__(self, name, figno, fig_type, plotdata):
         self._show = True
         self.add_attribute('plotaxes_dict', {})
         self.add_attribute('type',fig_type)   # = 'each_frame' or 'each_run' or 'each_gauge'
+        self.add_attribute('use_for_kml',False)
+        self.add_attribute('kml_dpi',200)
+        self.add_attribute('kml_xlimits',None)
```

Comment:
As to the `auto` problem, I am a bit disturbed that matplotlib changed its mind half way through.  How different is it?  For the record, I don't think this should happen regardless as the full domain of the solution should always be completely plotted, I guess it could be adding more buffer between the plot and the data?